controller/Review.php

```diff
@@ -100,68 +104,88 @@ public function init(): void
 
         $params = $this->getPsrRequest()->getQueryParams();
 
-        if (isset($params['serviceCallId'])) {
-
-            /** @var DeliveryExecutionFinderService $finder */
-            $finder = $this->getServiceLocator()->get(DeliveryExecutionFinderService::SERVICE_ID);
-
-            $data = $dataBuilder->create()->build($params['serviceCallId'], $finder->getShowScoreOption($this->ltiSession->getLaunchData()));
+        try {
+            if (isset($params['serviceCallId'])) {
```

Comment:
what do you think if we modify this condition by checking if the param is not empty
```suggestion
            if (!empty($params['serviceCallId'])) {
```